src/test/java/seedu/address/storage/JsonAdaptedCommentTest.java

```diff
@@ -0,0 +1,48 @@
+package seedu.address.storage;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.model.fields.Comment;
+
+public class JsonAdaptedCommentTest {
```

Comment:
You covered different aspects of JsonAdaptedComment class, including constructors, conversion methods, and exception handling. Well done!